content/en/docs/tasks/configure-pod-container/pull-image-private-registry.md

```diff
@@ -198,6 +198,73 @@ kubectl apply -f my-private-reg-pod.yaml
 kubectl get pod private-reg
 ```
 
+## Differences between Kubernetes and Docker
+
+The interpretation of `config.json` varies between the original Docker
+implementation and the Kubernetes interpretation. In Docker, the `"auths"` keys
```

Comment:
nit: no need to double-quote here.  `auths` is probably fine.

Reply:
Fixed as suggested :+1: